tests/mysql/generator_test.go

```diff
@@ -309,6 +312,36 @@ func newActorTableImpl(schemaName, tableName, alias string) actorTable {
 }
 `
 
+var tableSetSchemaFile = `
+//
+// Code generated by go-jet DO NOT EDIT.
+//
+// WARNING: Changes to this file may cause incorrect behavior
+// and will be lost if the code is regenerated
+//
+
+package table
+
```

Comment:
Go doc comment to describe what this function does.

Reply:
Resolved in 04c1a51